widget-function/widgetfn/handler.py

```diff
@@ -19,9 +20,14 @@ def get_widget(event, context={}):
 
     cloudwatch = client("cloudwatch")
     widget_img = cloudwatch.get_metric_widget_image(MetricWidget=widget_def)
+    encoded_image = base64.b64encode(widget_img["MetricWidgetImage"])
 
     res["statusCode"] = 200
-    res["headers"]["Content-Type"] = "image/png"
-    res["body"] = base64.b64encode(widget_img["MetricWidgetImage"])
-    res["isBase64Encoded"] = True
+    if event["queryStringParameters"]["asstring"] == "true":
```

Comment:
Any concerns with potential key errors here and similar lines below? Haven't perused the code at large yet so perhaps it'd be handled somewhere higher up anyway

Reply:
hit that key error when writing the test haha

Reply:
🧪